### ISSUE TITLE
Cannot import from *.css.js files: module has no exports (Reintroduced bug)

### ISSUE THREAD

**Issue by roginfarrer** [vanilla-extract-css/vanilla-extract#963]
### Describe the bug

I believe [this bug](https://github.com/vanilla-extract-css/vanilla-extract/issues/118), which was fixed when the `@vanilla-extract/babel-plugin` was required to use the webpack plugin, was reintroduced in `@vanilla-extract/webpack-plugin@2.2.0`. I was able to reproduce the issue in the codesandbox provided in that issue just by updating the webpack config to use the latest version of the webpack plugin and omitting the babel plugin.

I could resolve the issue by reintroducing the babel plugin and downgrading `@vanilla-extract/webpack-plugin` to `2.1.12`.

### Reproduction

https://codesandbox.io/s/cannot-import-from-css-js-files-module-has-no-exports-ct6ryh?file=/src/test.css.js:0-208

### System Info

```shell
System:
    OS: Linux 5.4 Debian GNU/Linux 10 (buster) 10 (buster)
    CPU: (16) x64 Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz
    Memory: 1.93 GB / 62.72 GB
    Container: Yes
    Shell: 5.0.3 - /bin/bash
  Binaries:
    Node: 12.22.12 - ~/.nvm/versions/node/v12.22.12/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v12.22.12/bin/yarn
    npm: 6.14.16 - ~/.nvm/versions/node/v12.22.12/bin/npm
  npmPackages:
    @vanilla-extract/css: 1.9.2 => 1.9.2
    @vanilla-extract/sprinkles: ^0.2.0 => 0.2.0
    @vanilla-extract/webpack-plugin: 2.2.0 => 2.2.0
    webpack: ^5.36.1 => 5.36.1
```

### Used Package Manager

yarn

### Logs

```shell
WARNING in ./src/App.tsx 71:25-40
export 'test' (imported as 'testStyles') was not found in './test.css' (module has no exports)
```

### Validations

- [X] Check that there isn't [already an issue](https://github.com/vanilla-extract-css/vanilla-extract/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by m7yue** [comment]
the same error i use in vite, how to resolve it!

**Comment by roginfarrer** [comment]
Fixed by #970 

You'll need to make sure that you're version of `@vanilla-extract/integration` is up-to-date. You may need to upgrade manually.
